methods_manuscript/main.md

```diff
@@ -111,11 +111,11 @@ This research was supported by the Gordon and Betty Moore Foundation’s Data-Dr
 
 Figure 1: Flowchart of initial model building and automated pipeline steps. Letters indicate the associate steps discussed in the main text. 
 
-Figure 2: Screenshot of the forecast presentation website (http://phenology.naturecast.org) showing the February 21, 2019 forecast for the leaf out of *Acer saccharinum*. In the upper right is the interface for selecting different species, phenophases, or forecast issue dates via drop down menus.
+Figure 2: Screenshot of the forecast presentation website (http://phenology.naturecast.org) showing the forecast for the leaf out of *Acer saccharinum* in Spring, 2019, issued on Feburary 21, 2019. The maps represent the predicted date of leaf out (A), the anomaly compared to prior years (B), and the 95% confidence interval (C). In the upper right is the interface for selecting different species, phenophases, or forecast issue dates via drop down menus (D).
 
-Figure 3: Locations of phenological events which have occured between Jan. 1, 2019 and May 5, 2019 obtained from the USA National Phenology Network (blue circles), and all sampling locations in the same dataset (red points). Six individual plants are highlighted, where the solid line indicates the predicted event date as well as 95% uncertainty for a specified forecast issue date, and the dotted line indicates the observed event date. 
+Figure 3: Locations of phenological events which have occurred between Jan. 1, 2019 and May 5, 2019 obtained from the USA National Phenology Network (blue circles), and all sampling locations in the same dataset (red points). Four individual plants are highlighted, where the solid line indicates the predicted event date as well as 95% confidence interval for a specified forecast issue date, and the dashed line indicates the observed event date. The x-axis corresponds to the date a forecast was issued, while the y-axis is the date flowering or budburst was predicted to occur. For example: on Jan. 1, 2019 the *P. tremuloides* plant was forecasted to flower sometime between March, 29 and April, 24 (solid lines), while the actual flowering date was March 18 (dashed line).
```

Comment:
"forecast" instead of "forecasted" (both are technically valid for the past tense, but "forecast" is more common).

I'd drop the "while" and make the actual flowering date it's own sentence since there are a lot of commas and a parenthetical to read through already.